### PR TITLE
Add qps and burst options

### PR DESCRIPTION
**Pull request by alwaysdark**
increase client qps #44

### CODE REVIEW

File: cmd/app/options/options.go [hunk 1]
```diff
@@ -32,4 +34,6 @@ func NewServerOption() *ServerOption {
 
 func (s *ServerOption) AddFlags(fs *flag.FlagSet) {
 	fs.StringVar(&s.KubeConfig, "kubeconfig", "", "the path to the kube config")
+    fs.IntVar(&s.QPS, "qps", 5, "QPS indicates the maximum QPS to the master from this client.")
```
Comment (denkensk) [inline cmd/app/options/options.go:37]: go fmt this file

File: cmd/app/options/options.go [hunk 2]
```diff
@@ -23,6 +23,8 @@ import (
 // ServerOption is the main context object for the queue controller.
 type ServerOption struct {
 	KubeConfig string
+	QPS int
```
Comment (denkensk) [inline cmd/app/options/options.go:26]: Add some comments to these two variables

**Review by denkensk — COMMENTED** [review]
Thanks
Some small comments
@alwaysdark

**Comment by denkensk** [comment]
/lgtm
/approve
Thanks @alwaysdark

**Comment by alwaysdark** [comment]
ok

**Pull request merged**